docs/sphinx/user_guide/feature/loop_basic.rst

```diff
@@ -45,29 +45,22 @@ Simple Loops (RAJA::forall)
 ---------------------------
 
 As noted earlier, a ``RAJA::forall`` template executes simple 
-(e.g., non-nested) loops. For example, a C-style loop like::
+(e.g., non-nested) loops. For example, a C-style loop that adds two vectors,
+like::
 
-  double* a = ...;
-  double* b = ...;
-  double* c = ...;
-  
   for (int i = 0; i < N; ++i) {
     c[i] = a[i] + b[i];
   }
 
-may be written in a RAJA form as::
+may be written using RAJA as::
 
-  double* a = ...;
-  double* b = ...;
-  double* c = ...;
-  
-  RAJA::forall<exec_policy>(iter_space I, [=] (index_type i) {
+  RAJA::forall<exec_policy>(RAJA::RangeSegment(0, N), [=] (int i) {
```

Comment:
Why the switch from `index_type` to `int`?

Reply:
Since index_type isn't defined, I thought it simplifies the example.

Reply:
That makes sense. I just remembered we had problems with using int before with slowdowns from type casts.